app/assets/javascripts/admin/addon/templates/web-hooks.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="web-hooks-listing">
   <p>{{i18n "admin.web_hooks.instruction"}}</p>
   <div class="new-webhook">
-    <LinkTo @route="adminWebHooks.show" @model="new" @tagName="button" @classNames="btn btn-default">
```

Comment:
Do we really wanna remove `@tagName` here? (and in some other places)

Reply:
Yeah, we do. Using `<LinkTo>` with `@tagName` is deprecated. We should either use it like this or replace with `<DButton>` + new actions in related controllers.

Reply:
I see, makes sense 👌 

However, doesn't this mean that this is a behaviour change? Should we check each one of these instances and make sure that the button/link still looks ok, and still functions correctly?

Reply:
Done. Added a small css fix to fix spacing in one of the templates.